code/modules/paperwork/paper.dm

```diff
@@ -250,6 +250,7 @@
 
 
 /obj/item/paper/Topic(href, href_list)
+	return
```

Comment:
Could you add a comment to this line (and the other line that this PR modifies) that clarifies that this early return is intentional, so that people won't mistake it for shitcode and write issue reports about it?

Mentioning where the other early return is in that comment would be nice as well, so that someone won't forget to remove it when/if they remove this return.